components/PastEventsCarousel.jsx

```diff
@@ -0,0 +1,95 @@
+import Image from 'next/image';
+import { SwiperSlide } from 'swiper/react';
+import Tile from '../components/Tile';
+import TextSection from './TextSection';
+import Carousel from '../components/Carousel';
+import { PAST_EVENTS } from '../data/events';
+import styles from '../components/RolesCarousel.module.css';
+import Heading from "./Heading";
+import { UnderlineTypes } from "../utils/underlineType";
+import Link from "next/link";
+import 'swiper/css/navigation';
+
+const CAROUSEL_CONTAINER = styles.swiperContainer;
+const TILE_CONTAINER = 'flex flex-col lg:h-[650px] md:flex-column rounded-[1.875rem] p-[1.875rem]';
+const TILE_TEXT_CONTAINER = 'basis-3/5 md:basis-3/5 lg:basis-3/5 text-center md:py-10 ';
+const TILE_TEXT_SPACING = 'lg:space-y-48 py-5';
+const ARROW_INIT = { left: 'swiper-button-prev', right: 'swiper-button-next' };
+const LEFT_NAVIGATION_ARROW = styles.swiperButtonPrev;
+const RIGHT_NAVIGATION_ARROW = styles.swiperButtonNext;
+
+const PAST_EVENTS_CONTAINER = 'bg-[#00D3A9] pb-10';
+const PAST_EVENTS_CONTENT = 'px-10 md:px-24 lg:px-48 py-20';
+const PAST_EVENTS_HEADER = "flex flex-row";
+
+const LONG_SQUIGGLY_LINE_SVG = "/svgs/events/long_events_squiggly_line.svg";
+
+
+const TILE_IMAGE_CONTAINER = "flex flex-row justify-center md:flex-row md:justify-around lg:justify-center relative";
+
+
+const PastEventsTile = ({ event }) => (
+  <Tile classes={TILE_CONTAINER}>
```

Comment:
could you round the corners a little more? In the ticket the corners look a bit more rounded